doc/vic20.sgml

```diff
@@ -188,7 +188,7 @@ No VIC1011 drivers are currently available for the VIC20.
 
 <sect1>Escape code<p>
 
-For an Esc, press CTRL and the <tt/[/ key.
+For an Esc, pressing CTRL and the <tt/[/ key is not supported.
```

Comment:
> In case I'm right, I think the text should rather read "Entering an ESC is not possible." or alike.

It's "worse" than that.  The VIC-20 doesn't have a traditional control key.  CTRL-R is the only traditional key combo that produces anything.  On the VIC-20, the CTRL key is used with the number keys to change the cursor's color.

So ...
that "Escape code" subsection should be removed; and, a note should be put in the "Limitations" section.  It should say that the `CTRL` key can't be used to type most control characters.